tomcat-base/src/shared/java/eu/xenit/alfresco/tomcat/embedded/config/DefaultConfigurationProvider.java

```diff
@@ -0,0 +1,41 @@
+package eu.xenit.alfresco.tomcat.embedded.config;
```

Comment:
it's a little bit confusing to have some defaults in java, some in shell scripts. Can't we have it in the same place, like e.g. default alfresco-global.properties which is loaded first and then the env vars are used to override it ?

Reply:
which properties are in a shell script, other then the JAVA_OPTS ?

Reply:
JAVA_OPTS need to be done before application start. I try to keep those props to a minimum.